test/s3-resumption/s3-insert-long-gzip.td

```diff
@@ -8,6 +8,7 @@
 # by the Apache License, Version 2.0.
 
 $ s3-put-object bucket=test key=s3/a.gzip compression=gzip
+SRB66R9YUZQLH22X2LXNXF9MBSPK9GRMLEBATCYKKRPP0JY553EGWCSR5WJ44JJGBS0G77HEV3S4QGQ1CB9SXY2IBNQS8KZGCHUC5ALHC5F1YU1RJ0SKT44WBC62QO576EDOYP8EEUVSWWGTOOB0USDW07O0IM0GZA0KPHU5K3SSP6V5SHGZK26AV7EH8JIOIOSJSYSASDABH7SNI4E72ZWMQTL239P3G2HT55DTRF94OD2DWAZM5XZKTXTKVMGV30NDXZD3WS67TY81QT0S3EKTQRPSVUOEQKDOCMTO74MGSRT0KQ4LFZ442V9LAXDS2CIZCNID6Z02IBC8O9K1QJIHYOMJIO9PP8HFLH1LG9YOS72SQ8VE5RDZKMC1UAG6BZ0WWQMZI22I8IRUOTZ5RUZOQQYFLNMQLQ9DNWMFNBKMPB2BWKWXCKWHY3VHCRCTGEO6IGHJS9FXZCQIIC7R3KGMG4LW57OWM6NS95AEZIZDC181MULUJ7M6XNTD1DSV7R5KLKQ7DGF2JIB3JOOHRIUBS7TLEDEMBTXF1R9OPZO3GSB6XJR18ZYLH40RXZD8HYX76JQ9HQ37Z4QFMMBIGI9XO45VJ5UMBCJG8B54MU9W2U19A5UY96R3O034GQ9D7A0HIMNRGAIH9U2Z13KCFK9Y6N7SQJV5A72E5J1TFOC714W04WQS4UGQVH2FZH5SC8Z6RZDFSMU3R8LD3N9T5KT1IMOQSSULX6NLBVVVG44HWCTK0NNHDC9BQB007PP7NUJ7S56ANGJUAV5R0QH3LN2L3U6JLY5T01JHFX55YKCKARP7VNF812VZH4Z56H3OOQ52P1XIXBR1IUE9EQDTKCCX5NIP7GTV8TQDZYXWVPNVU9PJTO1J2DAG7K8ZOLCCC0R8HV2DBRISMMXOLA4XT88S3OJXF9KOJRE5RCAE5352V23MQPDGTPOSVWYMPN433FBEORKX4DMX4UR6I4VMVKE08FXI4XETRP9F8J8PHCFAV2H41URQDWKX8ORSZ5Y1
```

Comment:
Do we need the trivially compressible section afterwards?

Also, can we make this long line nicely formatted at the 100char mark?

Reply:
Yes, I actually think the trivially compressible section is more realistic, so I think it should stay.

I have not thought about wrapping the non-compressible section, but now that I am forcing myself to think arguments against  wrapping caused by my laziness , actually by not wrapping it we force it to be ingested as a single row , which I think also adds some variety to the test. I even thought about adding a 1Mb row, but then it would have strained all the  git and UI tools it ever passed through.

Reply:
we could add a testdrive command to s3-generate an object, we already have basically all the code needed for it in `test/performance/s3-datagen`